SlothfulSheep/bangazon-site
# Delete Product Listing

# Description

Users are able to click on **_My Products_** and see a list of products they have listed for sale. They can then delete the product.

Fixes # (issue)

## Type of change


# How Has This Been Tested?

1. Run `npm start` from the appropriate directory.
2. Login as an existing user and click **Sell a Product** from the nav, then add a product.
3. Click on **My Products**, the users product should appear.
4. Click Delete. The product should be removed and the page should reload.

# Checklist:
